airgun/entities/activationkey.py

```diff
@@ -32,23 +36,29 @@ def read(self, entity_name):
 
     def update(self, entity_name, values):
         view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        view.flash.assert_no_error()
```

Comment:
you should assert and dismiss alerts after update, not before, as there won't be any notifications before update :)